pkg/kubectl/cmd/run.go

```diff
@@ -83,7 +86,8 @@ func NewCmdRun(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *c
 }
 
 func addRunFlags(cmd *cobra.Command) {
-	cmd.Flags().String("generator", "", "The name of the API generator to use.  Default is 'run/v1' if --restart=Always, otherwise the default is 'run-pod/v1'.")
+	// TODO: Change the default to "deployment/v1beta1" when deployment reaches beta (#15313)
```

Comment:
Now we set `kubectl run` default to create replication controllers (`--generator=run/v1`) when restartPolicy == api.RestartPolicyAlways. We want to make it default to create deployments (`--generator=deployment/v1beta1`) in the future once Deployment reaches beta. Therefore, this TODO needs to stay.